src/cli/onefuzz/api.py

```diff
@@ -1439,7 +1439,7 @@ def create(
                 vm_sku=vm_sku,
                 image=image,
                 region=region,
-                size=size,
+                size=1,
```

Comment:
It would be good to explain _why_ we set the size to 1. Probably not going to remember later.

Reply:
I'll add a comment.